pages/common/middleman.md

```diff
@@ -0,0 +1,28 @@
+# middleman
+
+> Static site generator written in Ruby.
+> More information: <https://middlemanapp.com/>.
+
+- Install the Middleman gem:
```

Comment:
That's not a middleman command. Let's remove this.

Reply:
You are right. I am fixing that.